packages/kinvey-nativescript-sdk/package.json

```diff
@@ -9,7 +9,7 @@
   "main": "kinvey-nativescript-sdk.js",
   "typings": "kinvey.d.ts",
   "dependencies": {
-    "nativescript-sqlite": "~1.1.11",
+    "nativescript-sqlite": "latest",
```

Comment:
Do we use the `latest` for other dependencies? Why is this dependency different than others in terms of what version to use?

Reply:
Also, since the latest version of `nativescript-sqlite` is now at v2, do we have to check for any breaking changes?

Reply:
I am use the `latest` because this was what the README for using encryption said to do.

Reply:
I have not noticed any changes that break how we are using `nativescript-sqlite`

Reply:
v2 major change was in the way the plugin allowed other plugins to be loaded automatically such as the encryption plugin. There were no changes to the API for `nativescript-sqlite` according to the changelog